src/main/asciidoc/project/jta/introduction.adoc

```diff
@@ -1,24 +1,4 @@
 
 = Introduction
 
-Apart from ensuring that the run-time system is executing normally, there is little continuous administration needed for the {productName} software.
-Refer to <<jta_important_points_for_administrators>> for some specific concerns.
-
-[[jta_important_points_for_administrators]]
-.Important Points for Administrators
-* The present implementation of the {productName} system provides no security or protection for data.
-The objects stored in the {productName} object store are (typically) owned by the user who ran the application that created them.
-The Object Store and Object Manager facilities make no attempt to enforce even the limited form of protection that Unix/Windows provides.
-There is no checking of user or group IDs on access to objects for either reading or writing.
-* Persistent objects created in the Object Store never go away unless the StateManager.destroy method is invoked on the object or some application program explicitly deletes them.
-This means that the Object Store gradually accumulates garbage (especially during application development and testing phases).
-At present we have no automated garbage collection facility.
-Further, we have not addressed the problem of dangling references.
-That is, a persistent object, `A`, may have stored a Uid for another persistent object, `B`, in its passive representation on disk.
-There is nothing to prevent an application from deleting `B` even though `A` still contains a reference to it.
-When `A` is next activated and attempts to access `B`, a run-time error will occur.
-* There is presently no support for version control of objects or database reconfiguration in the event of class structure changes.
-This is a complex research area that we have not addressed.
-At present, if you change the definition of a class of persistent objects, you are entirely responsible for ensuring that existing instances of the object in the Object Store are converted to the new representation.
-The {productName} software can neither detect nor correct references to old object state by new operation versions or vice versa.
-* Object store management is critically important to the transaction service.
+The notions explained here are to be considered as extensions of what has already been presented in the xref:general_procedures/general_introduction.adoc#important_points_for_administrators[General Procedures' Administration section].
```

Comment:
I might be wrong but when I look at what was built having the reference like this seems to take me to basically a new (sub) document. I don't know how to make AsciiDoc pages aware of each other except in this manner but maybe  `<<important_points_for_administrators>>` works somehow?

Reply:
Yeah, I pushed it like this because I couldn't find another solution :confused:. `<<important_points_for_administrators>>` would have worked if the reference had been on the same adoc page.

Reply:
I think we might find one because I think I can see different adocs can reference each other

Reply:
This was fixed following Tom's suggestion to use `link:#...`. Thanks Tom

Reply:
Thank you!